src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_COST = 40;
+  const SHORT_TERM = 3;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM_DISCOUNT = 20;
```

Comment:
The use of constants for `RENT_COST`, `SHORT_TERM`, `LONG_TERM`, `LONG_TERM_DISCOUNT`, and `SHORT_TERM_DISCOUNT` is excellent as it avoids hardcoded values and makes the code more readable and maintainable.